doc/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.2"
+
+services:
+  vscode:
+    build: .
+    image: latexdevcontainer
+    volumes:
+      - ../:/workspace
+      # Docker socket to access Docker server
+      - /var/run/docker.sock:/var/run/docker.sock
+      # SSH directory
+      - ~/.ssh:/root/.ssh
+      # For Windows without WSL, a copy will be made
+      # from /tmp/.ssh to ~/.ssh to fix permissions
+      # - ~/.ssh:/tmp/.ssh:ro
+      # Shell history persistence
+      - ~/.zsh_history:/root/.zsh_history:z
+      # Git config
+      - ~/.gitconfig:/root/.gitconfig
+    environment:
+      - TZ=
+    entrypoint: ["zsh", "-c", "while sleep 1000; do :; done"]

```

Comment:
Can you explain why the while loop like this is in the entrypoint?

Reply:
This is to work around a bug from VS Code >= 1.64 (currently v1.73) when spinning up the containers using the extension. The work around was detailed in this issue on the VS Code's repository: https://github.com/microsoft/vscode-remote-release/issues/6337. From my understanding, it's a race condition that prevents the dev container from being gracefully started and VS Code to remote into the container.

Reply:
Thanks. It'd be good to add a comment to that effect in this yml file for those who are not in the know.

Reply:
Got it, I will add more details in the `.devcontainer/README.md` on my next push